src/__internal__/input/input-presentation.style.ts

```diff
@@ -6,11 +6,17 @@ import { InputContextProps } from "../input-behaviour";
 import { CarbonProviderProps } from "../../components/carbon-provider";
 
 export const StyledInputPresentationContainer = styled.div<
-  Pick<CommonInputPresentationProps, "inputWidth">
+  Pick<CommonInputPresentationProps, "inputWidth"> & {
+    validationRedesignOptIn?: boolean;
+  }
 >`
   flex: 0 0 ${({ inputWidth }) => inputWidth}%;
   display: flex;
-  position: relative;
+  ${({ validationRedesignOptIn }) =>
+    !validationRedesignOptIn &&
+    css`
+      position: relative;
+    `}
```

Comment:
The `ErrorBorder` needs to match the height of the `InputPresentation` and `ValidationMessage` which can now wrap so requires some updates to how all that was previously implemented. The `ErrorBorder` already has `position: absolute` so it's possible to achieve the requirements by:
- adding a new grandparent container with `position: relative`
- updating the border styles by removing `height: calc(100% + var(--spacing300))` and adding `top: 0px` 
- removing the position: relative on the `InputPresentation` container. 

When the `validationRedesignOptIn` isn't set the container needs to have `position: relative` relative again to allow the `ValidationIcon` to be positioned. I want to avoid making too many changes given that the `Textbox` is consumed in a lot of other components but there may be a better approach so happy to discuss this on slack if you want.

If the container has `position: relative` with validation redesign:
![image](https://user-images.githubusercontent.com/44157880/204030127-da3fdf95-6bd5-440f-8f4d-c9d1aacd61fa.png)

If the container does not have `position: relative` with validation redesign:
![image](https://user-images.githubusercontent.com/44157880/204030203-65f1bc9c-ce93-4614-82da-4ca6a80951e8.png)

Reply:
Ah I see, thanks - not sure how I missed the difference with the length of the red border. Quite happy with your solution, my comment was only because I'd missed that this actually had any effect!